lib/logic/models/album.dart

```diff
@@ -37,20 +37,15 @@ class Album extends PersistentQueue implements PlatformAlbum {
   bool get playable => true;
 
   /// Gets album normalized year.
-  int get year {
-    return lastYear == null || lastYear! < 1000 ? clock.now().year : lastYear!;
+  int? get year {
+    return lastYear ?? firstYear;
```

Comment:
I don't think using `clock.now().year` makes sense.

We could potentially add some logic to convert `0 <= year < 50` to `2000 + year` and `50 <= year < 100` to `1900 + year`, but I'm not sure if that's a good long term solution to have that.

And I think falling back to `firstYear` makes sense either way.